lib/core/src/lib/card-view/components/card-view/card-view.component.stories.ts

```diff
@@ -0,0 +1,244 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+import { CardViewComponent } from './card-view.component';
+import { CoreStoryModule } from './../../../testing/core.story.module';
+import {
+    CardViewArrayItemModel,
+    CardViewBoolItemModel,
+    CardViewDateItemModel,
+    CardViewDatetimeItemModel,
+    CardViewIntItemModel,
+    CardViewKeyValuePairsItemModel,
+    CardViewMapItemModel,
+    CardViewModule,
+    CardViewSelectItemModel,
+    CardViewTextItemModel
+} from '../../../../..';
+import { of } from 'rxjs';
+
+export default {
+    component: CardViewComponent,
+    title: 'Core/Card View/Card View',
+    decorators: [
+        moduleMetadata({
+            imports: [CoreStoryModule, CardViewModule]
+        })
+    ],
+    argTypes: {
+        editable: {
+            control: 'boolean',
+            defaultValue: true
+        },
+        displayEmpty: {
+            control: 'boolean',
+            defaultValue: true
+        },
+        displayNoneOption: {
+            control: 'boolean',
+            defaultValue: true
+        },
+        displayClearAction: {
+            control: 'boolean',
+            defaultValue: true
+        },
+        copyToClipboardAction: {
+            control: 'boolean',
+            defaultValue: true
+        },
+        useChipsForMultiValueProperty: {
+            control: 'boolean',
+            defaultValue: true
+        },
+        multiValueSeparator: {
+            control: 'text',
+            defaultValue: ', '
+        }
+    }
+} as Meta;
+
+const dataSource = [
```

Comment:
Let's keep the mock data in a separate files like this as advised by Tomek: 
![image](https://user-images.githubusercontent.com/64163335/191740473-7bd883f7-212f-4093-821a-b3717a38beb6.png)
![image](https://user-images.githubusercontent.com/64163335/191740546-1a28603c-b9a2-4d7f-a5c4-4be5f46f5b6b.png)
![image](https://user-images.githubusercontent.com/64163335/191740587-155b0000-a38c-4f4b-b334-032f283b3171.png)

Reply:
Done in f5b24e0edadf13a3dc5f18cc561e292ab81a3e09